content/src/service/deployments/DeploymentManager.ts

```diff
@@ -173,6 +176,49 @@ export class DeploymentManager {
     }
   }
 
+  async newGetPointerChanges(
+    components: Pick<AppComponents, 'database'>,
+    options?: PointerChangesOptions
+  ): Promise<PartialDeploymentPointerChanges> {
```

Comment:
This can be extracted to an unit-testable function to prevent passing on a dummy DeploymentManager class